torch/distributed/rpc/__init__.py

```diff
@@ -33,10 +33,15 @@ def is_available():
 
     import numbers
 
+    if sys.platform != 'win32':
+        from .options import TensorPipeRpcBackendOptions  # noqa: F401
+        _default_rpc_backend = BackendType.TENSORPIPE
```

Comment:
do we also aim to enable RPC on Windows? If so, shall we split this out into a separate PR? And if this is the case, we shouldn't disable `${TORCH_ROOT}/test/cpp/rpc` in that followup PR.